documentation/modules/ROOT/pages/operations/kubernetes.adoc

```diff
@@ -48,30 +48,20 @@ $ kubectl create ns debezium-example
 As mentioned above, for the {prodname} deployment we will use Strimzi, which manages the Kafka deployment on Kubernetes.
 Please see the https://strimzi.io/docs/operators/latest/deploying.html[Strimzi deployment documentation] for more details on how to deploy Strimzi on your Kubernetes cluster.
 
-For testing purposes, you can use the following, simplified and unsecured, approach.
-
-* First, https://strimzi.io/downloads/[download] the latest Strimzi release and unpack it into a local directory
-* Create a dedicated namespace for the Strimzi Kafka Cluster Operator:
-
-[source,subs="attributes",options="nowrap"]
-----
-$ kubectl create ns kafka
-----
-
-* Modify the downloaded source files to use the `kafka` namespace for the installation of the Strimzi Kafka Cluster Operator:
+The most simple way is to install Strimzi operator.
```

Comment:
So Strimzi really _is_ an operator, there's no way to use Strimzi without its operator. I'll reword this slightly:

> This simplest way for installing Strimzi is using the Operator Lifecycle Manager ...

That said, it's a nice improvement!